netlify.toml

```diff
@@ -2421,138 +2111,132 @@
     to = "/partners/altinity"
 
 
-# Added: 2022-10-02
-[[redirects]]
-    from = "/docs/integrate/client/browser-extension"
-    to = "/docs/integrate/browser-extension"
-
 # Added: 2022-10-11
 [[redirects]]
-    from = "/handbook/small-teams/experimentation"
+    from = "/handbook/people/team-structure/experimentation"
     to = "/handbook/small-teams/experimentation"
 
 
 # Added: 2022-10-11
 [[redirects]]
-    from = "/handbook/small-teams/pipeline"
+    from = "/handbook/people/team-structure/pipeline"
     to = "/handbook/small-teams/pipeline"
 
 # Added: 2022-10-11
 [[redirects]]
-    from = "/handbook/small-teams/product-analytics"
+    from = "/handbook/people/team-structure/product-analytics"
     to = "/handbook/small-teams/product-analytics"
 
 
 # Added: 2022-10-11
 [[redirects]]
-    from = "/handbook/small-teams/team-structure"
+    from = "/handbook/people/team-structure/team-structure"
     to = "/handbook/small-teams/team-structure"
 
 
 
 # Added: 2022-10-11
 [[redirects]]
-    from = "/handbook/small-teams/customer-success/mission"
+    from = "/handbook/people/team-structure/customer-success/mission"
```

Comment:
Just FYI on `/mission`, `/objectives`, and `/mission` urls, these are partials that are included on the small team pages. They're auto-generated but we don't really need redirects for them, since these pages are never hit directly.

Reply:
Got it 👍